components/Dialogs/examples/AlertComparison.swift

```diff
@@ -0,0 +1,142 @@
+      NSLayoutConstraint(item:materialButton,
+                       attribute:.centerY,
+                       relatedBy:.equal,
+                       toItem:self.view,
+                       attribute:.centerY,
+                       multiplier:1.0,
+                       constant: 0.0)
+      ])
+
+
+      UIKitButton.translatesAutoresizingMaskIntoConstraints = false
+      UIKitButton.setTitle("UIKit Alert", for: UIControlState())
+      UIKitButton.setTitleColor(UIColor(white: 0.1, alpha:1), for: UIControlState())
+      UIKitButton.sizeToFit()
+      UIKitButton.addTarget(self, action: #selector(tapUIKit), for: .touchUpInside)
+      self.view.addSubview(UIKitButton)
+
+      NSLayoutConstraint.activate([
+      NSLayoutConstraint(item:UIKitButton,
+      attribute:.centerX,
+      relatedBy:.equal,
+      toItem:self.view,
+      attribute:.centerX,
+      multiplier:1.0,
+      constant: 0.0),
+      NSLayoutConstraint(item:UIKitButton,
+      attribute:.top,
+      relatedBy:.equal,
+      toItem:materialButton,
+      attribute:.bottom,
+      multiplier:1.0,
+      constant: 8.0)
+        ])
+  }
+
+  @objc func tapMaterial(_ sender: Any) {
+    let messageString = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur " +
+    "ultricies diam libero, eget porta arcu feugiat sit amet. Maecenas placerat felis sed risus " +
+    "maximus tempus. Integer feugiat, augue in pellentesque dictum, justo erat ultricies leo, " +
+    "quis eleifend nisi eros dictum mi. In finibus vulputate eros, in luctus diam auctor in. "
+
+    let alertController = MDCAlertController(title: nil, message: messageString)
+
+    let acceptAction = MDCAlertAction(title:"Accept") { (_) in print("Accept") }
+    alertController.addAction(acceptAction)
+
+    let considerAction = MDCAlertAction(title:"Consider") { (_) in print("Consider") }
+    alertController.addAction(considerAction)
+
+    let rejectAction = MDCAlertAction(title:"Reject") { (_) in print("Reject") }
+    alertController.addAction(rejectAction)
+
+    self.present(alertController, animated: true, completion: nil)
+  }
+
+  @objc func tapUIKit(_ sender: Any) {
+    let messageString = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur " +
+      "ultricies diam libero, eget porta arcu feugiat sit amet. Maecenas placerat felis sed risus " +
+      "maximus tempus. Integer feugiat, augue in pellentesque dictum, justo erat ultricies leo, " +
+    "quis eleifend nisi eros dictum mi. In finibus vulputate eros, in luctus diam auctor in. "
+
+    let alertController = UIAlertController(title: nil, message: messageString, preferredStyle:.alert)
+
+    let acceptAction = UIAlertAction(title:"Accept", style:.default)
+    alertController.addAction(acceptAction)
+
+    let considerAction = UIAlertAction(title:"Consider", style:.default)
+    alertController.addAction(considerAction)
+
+    let rejectAction = UIAlertAction(title:"Reject", style:.default)
```

Comment:
No handlers implemented on these actions?

Reply:
Unused, but added for parity with the Material sample.